src/core/feature-flags/flags.dev.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
was adding this file intended? I dont see how its being refrenced

Reply:
Oh. It was accidently. Thank you!